vite-project-kabinet/src/Content/PaginatedList.jsx

```diff
@@ -32,7 +32,7 @@ export const PaginatedList = ({items, itemsPerPage}) => {
                     onClick={() => handlePageChange(currentPage - 1)}
                     disabled={currentPage === 1}
                 >
-                    <img src={arrowLeft} alt="šipka vlevo"/>
+                    <img src={arrowLeft} alt="arrow left"/>
```

Comment:
Detail: tohle jsi mohla názvy souborů se týkají vývojářů a je fajn to mít konzistentní. Alt text slouží uživatelům a tedy by dávalo smysl, aby byly ve slovenštině. Ačkoli v praxi je běžný uživatel neuvidí.